frontend-react/src/components/shared/Table/Table.module.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core' as *;
+
+.reportstreamTable {
+  :global {
+    // Since these are USDS overrides, we use the :global property so we can
+    // locally scope everything within the CSS Module underneath .reportstreamTable
+    // without having to create new SCSS classes, and simply made overrides.
+    .usa-table thead th {
+        cursor: pointer;
+        user-select: none;
+        background-color: color('gray-5');
+    }
+
+    .usa-table thead th.active-col-header {
+        background-color: color('blue-30');
+    }
+
+    .usa-table td.active-col-td {
+        background-color: color('blue-10');
+    }
+
+    .usa-table thead .column-header-title-container {
+        display: flex;
+        align-items: center;
+    }
+
+    .usa-table thead .column-header-title-container p {
+        font-size: 14px;
+        margin: 0;
+    }
+
+    .usa-table thead .column-header-title-container svg {
+        flex: none;
+        margin-left: 2px;
+        height: 28px;
+        width: 28px;
+    }
+
+    .usa-table td.message-receiver-break-word {
```

Comment:
This shouldn't be here, right?  It's a specific rule for the Message Tracker feature that should exist separately, not part of the generic table styles.

Reply:
Fixed all the CSS classes to be more specific / tailored to a generic component.